libsql-server/tests/common/net.rs

```diff
@@ -141,6 +141,8 @@ pub trait SimServer {
 #[async_trait::async_trait]
 impl SimServer for TestServer {
     async fn start_sim(mut self, user_api_port: usize) -> anyhow::Result<()> {
+        let _ = tracing_subscriber::fmt::try_init();
```

Comment:
there is a `init_tracing` function already somewhere already, and it's called in multiple tests

Reply:
Yeah, I just called it here just to make sure it gets called. We should probably refactor some of our test harness code a bit since this spot is a bit hacky.